ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -4,6 +4,30 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+trap 'FRC=$?; createUpgradeJunit' ERR EXIT TERM
+
+# Generate the Junit for upgrade
+function createUpgradeJunit() {
+    echo "Generating the Junit for upgrade"
+    if [[ "${FRC}" =  0  ]]; then
+      cat >"${ARTIFACT_DIR}/junit_upgrade.xml" <<EOF
+      <?xml version="1.0" encoding="UTF-8"?>
+      <testsuite name="cluster upgrade" tests="1" failures="0">
+        <testcase name="upgrade should succeed"/>
```

Comment:
Sorry, I missed here. Add attribute `classname="cluster upgrade"` so that the test case will be nested under the test suite. 

```
<testcase classname="cluster upgrade" name="upgrade should succeed"/>
```

Reply:
Thanks!! Updated.